src/foam/nanos/auth/AuthorizableAuthorizer.java

```diff
@@ -10,16 +10,16 @@
 import foam.core.X;
 
 public class AuthorizableAuthorizer implements Authorizer {
-  private static AuthorizableAuthorizer instance_ = null;
 
-  public static AuthorizableAuthorizer instance() {
-    if ( instance_ == null ) {
-      instance_ = new AuthorizableAuthorizer();
-    }
-    return instance_;
+  private String name;
```

Comment:
Rename to something more meaningful, lik permissionPrefix_. Make protected rather than private since private just makes testing and debugging more difficult.